bundles/CoreBundle/config/l10n.yaml

```diff
@@ -13,7 +13,6 @@ services:
             - '@Pimcore\Translation\Translator.inner'
         calls:
             - [setKernel, ['@kernel']]
-            - [setAdminPath, ['%pimcore.admin.translations.path%']]
             - [setAdminTranslationMapping, ['%pimcore.translations.admin_translation_mapping%']]
```

Comment:
we should also move `setAdminTranslationMapping` call to the translator compiler pass

Reply:
and maybe this as well https://github.com/pimcore/pimcore/blob/b2110bb3c416cbf4a7fce967fe98a0e031d642a0/bundles/CoreBundle/config/documents.yaml#L18